omero/sysadmins/unix/server-centos7-ice36.txt

```diff
@@ -87,14 +91,14 @@ Configure:
     :start-after: #end-release-ice36
     :end-before: #end-step04
 
-Installing a web server
------------------------
+Installing Nginx
+----------------
 
 OMERO.web is deployed using Nginx see :doc:`install-web/install-nginx`.
 
 **The following steps are run as root.**
 
-Install Nginx 1.8, copy the Nginx OMERO configuration file into the
+Install the latest stable version of Nginx (1.10), copy the Nginx OMERO configuration file into the
```

Comment:
true  I was thinking about that too. Initially I had a date next to it but it did not work
I will remove it to avoid confusion